seis/ms/btime.go

```diff
@@ -0,0 +1,90 @@
+package ms
+
+import (
+	"encoding/binary"
+	"time"
+)
+
+// BTimeSize is the fixed size of an encoded BTime.
+const BTimeSize = 10
+
+// BTime is the SEED Representation of Time.
+type BTime struct {
+	Year   uint16
+	Doy    uint16
+	Hour   uint8
+	Minute uint8
+	Second uint8
+	Unused byte
+	S0001  uint16
+}
+
+// Time converts a BTime into a time.Time.
+func (b BTime) Time() time.Time {
+	return time.Date(
+		int(b.Year),
+		1,
+		1,
+		int(b.Hour),
+		int(b.Minute),
+		int(b.Second),
+		int(b.S0001)*100000,
+		time.UTC,
+	).AddDate(0, 0, int(b.Doy-1))
```

Comment:
`b.Doy` or `b.Doy-1` ? please double check

Reply:
The `Doy` of a year starts at 1, so for the first Day (Jan 01)  we don't want to add 1 as that would then be Jan 02.